Controller/Adminhtml/CompanyCodes/Get.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * @category    ClassyLlama
+ * @copyright   Copyright (c) 2018 Classy Llama Studios, LLC
+ * @author      sean.templeton
+ */
+
+namespace ClassyLlama\AvaTax\Controller\Adminhtml\CompanyCodes;
+
+use ClassyLlama\AvaTax\Exception\AvataxConnectionException;
+use Magento\Framework\DataObject;
+
+class Get extends \Magento\Backend\App\Action
+{
+    protected $resultPageFactory;
+
+    /**
+     * @var \ClassyLlama\AvaTax\Framework\Interaction\Rest\Company
+     */
+    protected $company;
+
+    /**
+     * @var \ClassyLlama\AvaTax\Helper\Config
+     */
+    private $config;
+
+    public function __construct(
+        \Magento\Backend\App\Action\Context $context,
+        \Magento\Framework\Controller\Result\JsonFactory $resultPageFactory,
+        \ClassyLlama\AvaTax\Framework\Interaction\Rest\Company $company,
+        \ClassyLlama\AvaTax\Helper\Config $config
+    )
+    {
+        parent::__construct( $context );
+        $this->resultPageFactory = $resultPageFactory;
+        $this->company = $company;
+        $this->config = $config;
+    }
+
+    public function execute()
```

Comment:
@LordZardeck - Method needs a docblock.  (I'm not concerned that the docblock for a controller "execute" method have a description, but it at least needs to define e.g. return type.)